test/system/post_test.rb

```diff
@@ -122,6 +122,22 @@ def assert_page_does_not_reload(message = "page should not reload")
 
     # Make sure that image has been uploaded
     page.assert_selector('#preview-main img', count: 1)
+    
+    fill_in("title", with: "My wiki page")
+    fill_in("text-input-main", with: "All about this interesting stuff")
```

Comment:
hmm, can't get this to work...

```
Minitest::UnexpectedError:         Capybara::ElementNotFound: Unable to find visible field "text-input-main" that is not disabled
            test/system/post_test.rb:127:in `block in <class:PostTest>'
```

Reply:
Hmm. what's the issue here? https://rubydoc.info/github/jnicklas/capybara/Capybara/Node/Actions:fill_in says this should work!

Reply:
aha, we had been looking at the preview, so the textarea was hidden!